components/NavigationDrawer/BUILD

```diff
@@ -52,20 +49,11 @@ mdc_extension_objc_library(
 
 mdc_objc_library(
     name = "private",
+    testonly = 1,
     hdrs = native.glob(["src/private/*.h"]),
-    includes = ["src/private"],
     visibility = ["//visibility:private"],
 )
 
-mdc_objc_library(
-    name = "TestingHeader",
```

Comment:
I am not sure if this target is being used somewhere, maybe put some context it in the PR description?

Reply:
I couldn't have been used since removing it caused no problems.

Reply:
What context would you like in the description? This target wasn't used and was part of the "clean up".